packages/admin/src/table/filterbar/FilterBarActiveFilterBadge.tsx

```diff
@@ -0,0 +1,56 @@
+import { createStyles, Theme, Typography, WithStyles, withStyles } from "@material-ui/core";
+import { FieldState } from "final-form";
+import { isEmpty, isEqual } from "lodash";
+import * as React from "react";
+
+import { difference } from "./difference";
+
+export type CometAdminFilterBarActiveFilterBadgeClassKeys = "hasValueCount";
+
+export const dirtyFieldsCount = (fieldState?: FieldState<any>) => {
+    if (fieldState?.initial) {
+        const diff = difference<any, Record<string, any>>(fieldState?.value, fieldState?.initial);
+        return Object.keys(diff).length;
+    }
+    return Object.keys(fieldState?.value).length;
+};
+
+const styles = (theme: Theme) =>
+    createStyles<CometAdminFilterBarActiveFilterBadgeClassKeys, any>({
+        hasValueCount: {
+            backgroundColor: `${theme.palette.grey[100]}`,
+            textAlign: "center",
+            borderRadius: "4px",
+            height: "20px",
+            width: "17px",
```

Comment:
I think this should just have a padding instead of a fixed width. Otherwise the number is cut off, when there are multiple digits.